saleor/graphql/account/utils.py

```diff
@@ -510,3 +510,21 @@ def check_is_owner_or_has_one_of_perms(
     """
     if not is_owner_or_has_one_of_perms(requestor, owner, *perms):
         raise PermissionDenied(permissions=list(perms) + [AuthorizationFilters.OWNER])
+
+
+def retrieve_user_by_email(email):
```

Comment:
IMO it's shouldn't be in `graphql` path. Maybe it's should be in `/saleor/account/utils.py`. 
We should add some unit tests for this function.

Reply:
agree, fits better, thanks